src/parser/select-parser.ts

```diff
@@ -118,6 +121,8 @@ type ExtractTypeFromSelectExpression<
   ? A extends ExtractAliasFromStringSelectExpression<RA>
     ? ExtractTypeFromStringSelectExpression<DB, TB, RA, A> | undefined
     : never
+  : SE extends AliasedAggregateFunctionBuilder<any, any, infer O>
```

Comment:
Same here, also needs the callback version.

Reply:
Good catch! done.